style.css

```diff
@@ -0,0 +1,214 @@
+* {
+  margin: 0px;
+  padding: 0px;
+}
+
+body {
+  background-color: #fff;
+  width: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  padding: 30px 10%;
+  background-color: darkorange;
+  padding: 60px;
+}
+
+.nav-list {
+  display: flex;
+  display: inline-block;
+  padding: 0px 20px;
+  text-decoration: none;
+  color: #000;
+}
+
+.navbar {
+  list-style: none;
+  display: flex;
+}
+
+.nav-list:hover {
+  color: #000;
+}
+
+span1 {
+  color: #000;
+}
+
+ span {
+  color:rgb(247, 3, 3);
+ }
+
+.nav-list,
+.logo {
+  color: #fff;
+  font-size: 35px;
+  text-decoration: none;
+}
+
+.first-h1 {
+  color: #000;
+  text-align: center;
+  margin-top: 30px;
+  font-size: 40px;
+}
+
+.main-image {
+  width: 100%;
+  height: 650px;
+  margin: 0px;
+}
+
+.first-p {
+  color:#fff;
+  background-color: darkorange;
+  font-size: 30px;
+  text-align: center;
+  margin-bottom: 50px;
+  padding: 25px;
+}
+
+.pizza-1 {
+  width: 100%;
+  height: 600px;
+}
+
+.ingredients {
+  background-color: #000;
+  color:darkorange;
+}
+
+.Pizza-dough {
+  margin-left: 400px;
+  font-size: 50px;
+}
+
+.pizza-p {
+  margin-left: 260px;
+  font-size: 35px;
+}
+
+.list-ingredient {
+  margin-left: 450px;
+  font-size: 30px;
+  margin-top: 15px;
+  padding: 20px;
+}
+
+.procedures {
+  background-color: #ff8c00;
+}
+
+.dough {
+  text-align: center;
+  font-size: 50px;
+  background-color: lightgray;
+  color: #000;
+  padding: 10px;
+}
+
+.insight-words {
+  margin-left: 30px;
+  margin-bottom: 30px;
+  font-size: 20px;
+  color: #000;
+  padding: 20px;
+}
+
+
+
```

Comment:
Keeping codebase DRY improves its readability and understandability. Please avoid empty spaces or lines in your code. I think line 166 also needs some deletions.